src/fluids.md

```diff
@@ -22,7 +22,7 @@ u$ and the time-extrapolated form
 
 \begin{align}
     \label{eq:Next}
-    N^\*(u^{n+1}) = \sum_{j=1}^k a_j N(u^{n+1-j})
+    N^\*(u^{n}) = \sum_{j=1}^k a_j N(u^{n+1-j})
```

Comment:
The idea here is to mark the extrapolation with a `*`, indicating that the operation computes a quantity that is actually at `n+1`. We did the same in the referenced paper.

In hindsight it would have been more clear to write something like
```
N^*_{u^{n+1}}(u^n)
```
but would look very cumbersome on paper.

Reply:
Fair enough, but I believe by that logic it should be `F^\*(u^{n+1})` everywhere instead of `F^\*(u^n)`?

Reply:
@jandrej: I adjusted it as you suggested, LGTM now.